torch/nn/_functions/thnn/upsampling.py

```diff
@@ -169,6 +171,14 @@ def backward(ctx, ggI):
 class UpsamplingNearest2d(Function):
 
     @staticmethod
+    def symbolic(g, input, size=None, scale_factor=None):
+        if scale_factor is None:
+            scale_factor = 1.0
+        if size is not None and set(size) != set([None]):
+            warnings.warn("ONNX export failed on UpsamplingNearest2d because size is not supported")
```

Comment:
You should use the helper function here. There's an example in `torch/nn/_functions/dropout.py`. It will prevent your bug here, where you claim that export failed, but then control falls through to the top level of the function.

Reply:
Yeah, will use the helper function.